jsonformatter_test.go

```diff
@@ -39,43 +39,43 @@ func TestCustomWriter(t *testing.T) {
 				inputLevel:    logrus.TraceLevel,
 				expectedLevel: 10,
 				inputTime:     now,
-				expectedTime:  now.Unix(),
+				expectedTime:  now.UnixNano() / int64(1e6),
```

Comment:
We should verify it's the correct time, in this test is very difficult (also if it seems correct)

Reply:
I have added an assert in the already existing test. Is it ok?